petals-esb-deployer-impl/src/test/java/org/ow2/petals/deployer/utils/ModelDeployerImplTest.java

```diff
@@ -47,4 +49,17 @@ public void deployModel() throws Exception {
 
         verify(runtimeModelDeployerMock).deployRuntimeModel(any(RuntimeModel.class));
     }
+
+    @Test
+    public void deployModelWithMavenUrl() throws Exception {
+        System.setProperty("java.protocol.handler.pkgs", "org.ops4j.pax.url");
```

Comment:
Use https://github.com/stefanbirkner/system-rules to set the system property, the system environment is automatically restored when the test ends